addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/BigAssFanHandlerFactory.java

```diff
@@ -23,6 +26,13 @@
  * @author Mark Hilbush - Initial contribution
  */
 public class BigAssFanHandlerFactory extends BaseThingHandlerFactory {
+    private Logger logger = LoggerFactory.getLogger(BigAssFanHandlerFactory.class);
+
+    @Override
+    protected void activate(ComponentContext componentContext) {
```

Comment:
Thanks, @JueBag!

@martinvw, if the issue is the reference to bundleContext, I can replace that with the above call.  Just let me know which way to go.